test/internet/test-http-dns-fail.js

```diff
@@ -5,34 +5,34 @@
  */
 
 const common = require('../common');
-var assert = require('assert');
-var http = require('http');
+const assert = require('assert');
+const http = require('http');
 
-var hadError = 0;
+let hadError = 0;
 
 function httpreq(count) {
   if (1 < count) return;
 
-  var req = http.request({
+  const req = http.request({
     host: 'not-a-real-domain-name.nobody-would-register-this-as-a-tld',
     port: 80,
     path: '/',
     method: 'GET'
   }, common.fail);
 
-  req.on('error', function(e) {
+  req.on('error', common.mustCall((e) => {
     console.log(e.message);
```

Comment:
I think this should be removed? I still don't know if we have an official policy on this or not ...